runtime/common/src/xcmp/mod.rs

```diff
@@ -1,6 +1,349 @@
 //! proposed shared XCM setup parameters and impl
+use crate::{AccountId, Balance};
+use codec::{Decode, Encode};
+use composable_traits::{
+	assets::{RemoteAssetRegistry, XcmAssetLocation},
+	oracle::MinimalOracle,
+};
+use frame_support::{
+	dispatch::Weight,
+	ensure, log, parameter_types,
+	traits::{Contains, Get},
+	weights::WeightToFeePolynomial,
+};
+use num_traits::Zero;
+use orml_traits::location::Reserve;
+use polkadot_primitives::v1::Id;
+use primitives::currency::{CurrencyId, WellKnownCurrency};
+use sp_runtime::traits::Convert;
+use sp_std::marker::PhantomData;
+use xcm::{latest::MultiAsset, prelude::*};
+use xcm_builder::*;
+use xcm_executor::{
+	traits::{FilterAssetLocation, ShouldExecute, WeightTrader},
+	*,
+};
 
-use frame_support::{dispatch::Weight, parameter_types};
 parameter_types! {
 	pub const BaseXcmWeight: Weight = 100_000_000;
 }
```

Comment:
Can we add a comment here describing why we chose this weight?

Reply:
will add. in short, this is taken from acala/hydra.

Reply:
weight of messages it self is added on top. it also kind of arbitrary. also once I saw mention of weights for XCMP, but never seen any benches in any chain